src/options/optionsMain.js

```diff
@@ -154,8 +154,8 @@ function optionsFactory(inpOptions, callback) {
 
 function sanitizeFilename(str) {
     const cleaned = sanitizeFilenameLib(str);
-    // remove non ascii
-    return cleaned.replace(/[^\x00-\x7F]/, '');
+    // remove all non ascii or do nothing
+    return cleaned.replace(/[^\x00-\x7F]/g, '') || cleaned;
```

Comment:
Sorry, I'm not really sure why do we have to "do nothing", wouldn't the `/g` flag be sufficient for removing the ascii characters?